File: projects/m1/010-month-name-to-number-of-days/solutions/jimmyhu/solution-month-name-to-number-of-days.py
long_month = ('january','march','may','july','august','october','dicember')
medium_month = ('april','june','september','november')
special_month = 'february'
mese = input('please insert the month(en): ').lower()
if mese in long_month:
    print('this month have 31 days')
elif mese in medium_month:
    print('this month have 30 days')
elif mese == special_month:
    year = int(input('please insert the year:'))
    if year % 4 == 0:
        print('than this month have 29 days')
    else:
        print('this month have 28 days')

File: projects/m1/010-month-name-to-number-of-days/solutions/valentinaBazzacchi/solution-month-name-to-number-of-days.js
// Dichiaro una variabile che avrà come valore l'anno corrente
let currentTime = new Date();
let year = currentTime.getFullYear();

// Dichiaro la variabile che assumerà il valore stringa inserito dall'utente e trasformo i caratteri in minuscolo
let month = prompt("Insert a month");
month = month.toLowerCase();

switch (month) {
/* Per il mese di febbraio imposto delle condizioni per cui verrà valutato il valore dell'anno corrente, 
se divisibile per 400 o per 4 ma non per 100 allora sarà bisestile */
    case "february":
        if (year%400==0){
        alert ("29 days");
        } else if (year % 4 == 0 && year % 100 != 0){
        alert ("29 days");
        } else {
        alert("28 days");
        }
    break;
        
    case "november":
    case "april":
    case "june":
    case "september":
    alert("30 days");
    break;
        
    case "january":
    case "march":
    case "maj":
    case "july":
    case "august":
    case "october":
    case "december":
    alert("31 days");
    break;
        
    default:
    alert("Invalid Entry!");
}